documentation/asciidoc/topics/proc_allocating_storage.adoc

```diff
@@ -1,12 +1,26 @@
-[id='container-storage_{context}']
-= Storage resources
-By default, {ispn_operator} allocates `1Gi` for storage for both {cacheservice} and {datagridservice} nodes.
-You can configure storage resources for {datagridservice} nodes but not {cacheservice} nodes.
+[id='allocating-storage_{context}']
+= Allocating storage resources
+You can allocate storage for {datagridservice} nodes but not {cacheservice} nodes.
 
+By default, {ispn_operator} allocates `1Gi` for the persistent volume claim.
+However you should adjust the amount of storage available to {datagridservice} nodes so that {brandname} can preserve cluster state during shutdown.
+
+[IMPORTANT]
+====
+If available container storage is less than the amount of available memory, data loss can occur.
+====
+
+.Procedure
+
+. Allocate storage resources with the `spec.service.container.storage` field.
+. Configure the `ephemeralStorage` and `storageClassName` fields as required.
```

Comment:
@oraNod probably I missed context, but both `ephemeralStorage` and `ephemeralStorage` are optional for `spec.service.container.storage` depends customer needed